partners/project-18.md

```diff
@@ -0,0 +1,59 @@
+<h2>Date</h2>
+<p>Fall 2017</p>
+<h2>Partner departments</h2>
+<ul>
+  <li>Health Canada (HC)</li>
+  <li>Global Affairs Canada (GAC)</li>
+</ul>
+<h2>Why this work matters</h2>
+<ul>
+  <li>Flu cases result in nearly 12,000 hospitalizations in Canada each year</li>
+</ul>
+<div class="row mrgn-tp-lg mrgn-bttm-lg">
+  <div class="col-md-8">
+    <div class="panel panel-success">
+      <header class="panel-heading">
+        <h4 class="panel-title text-center">Task success</h4>
+      </header>
+      <table class="table">
+        <thead>
+          <tr style="">
+            <th scope="col" class="col-md-3">Before</th>
+            <th scope="col" class="col-md-3">After</th>
+            <th scope="col" class="col-md-6">Improvement to task success</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td class="table-smnum">53%</td>
+            <td class="table-smnum">84%</td>
+            <td class="table-smnum"><span class="text-success"><strong>+31%pts</strong></span></td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</div>
+<h2>Task scenarios tested and improved</h2>
+<ul class="lst-spcd">
+  <li>This year's flu season could be worse than usual. Find the government’s advice on the most important way to avoid getting the flu.</li>
+  <li>Your friend put up a post urging her friends not to vaccinate their kids because of the risk of autism. Find some answers that would make you feel ready to discuss it with her.</li>
+  <li>You got a few mosquito bites on your honeymoon down south but you feel fine. Before getting pregnant, you want to make sure you don’t have Zika. Is there a test that can tell if you're infected?</li>
+  <li>You plan to start a family soon, but want a sunny vacation first. Which of the countries below should you avoid because of the risk of Zika?</li>
+  <li>Your baby was born on the first of this month. At what age is the first set of vaccinations recommended?</li>
+  <li>It is flu season and a family member is pregnant. Find out whether or not she should get a flu shot while pregnant.</li>
+  <li>You don't want to miss work tomorrow but should stay away if you have the flu. Yesterday, all of the symptoms below hit: find out which ones are the signal that it's the flu.</li>
+  <li>You think you heard something on the news about people being sick in Italy. Is there a particular concern you should be aware of before you go with your baby?</li>
+  <li>You are planning a trip to Mexico. Which specific vaccinations are recommended before you go?</li>
+  <li>The weekly flu report says most cases are an H3N2 flu. Find out if that particular flu is in this year's vaccine in Canada to help persuade your partner to get a flu shot.</li>
+</ul>
+<h2>Research summary</h2>
+<p><a href="https://blog.canada.ca/research-summaries/diseases-research-summary.html">GST/HST and payroll</a></p>
```

Comment:
This is incorrect. The link text should be: Diseases and immunization information" and the link should point to: https://blog.canada.ca/research-summaries/diseases-research-summary.html

Reply:
Fixed.